spec/wprof_spec.rb

```diff
@@ -0,0 +1,5 @@
+RSpec.describe Wprof do
```

Comment:
It would be nice to add more unit testing before we had an estable version.

Reply:
Yes know, i was thinking make that in the next PR (in this we already have a lot for process). However, if I can push something, i will.

Reply:
Don't worry make it in another PR 😄